README.md

```diff
@@ -1,6 +1,81 @@
 # xcvbnm
---------
+
 This is intended as a Dart port of zxcvbn (see https://github.com/dropbox/zxcvbn), a realistic password strength 
 estimator.
 
 **Please note, that this is currently in development.**
+
+## usage
+
+    
+    library xcvbnm.example.bin;
+    
+    import 'package:xcvbnm/xcvbnm.dart';
+    
+    test1() {
+      String password = "my_pwd_1";
+      // Use the direct helper API
+      Result result = xcvbnm(password);
+      print(result.toMap());
+    }
+    
+    test2() {
+      String password = "my_pwd_1";
+      // Use the object (can allow mocking)
+      Xcvbnm xcvbnm = new Xcvbnm();
+      Result result = xcvbnm.estimate(password);
+      print(result.toMap());
+    }
+    
+    main() {
+      test1();
+      test2();
+    }
+
+    
+### mock
+
+Mock Xcvbnm can be mocked. there is a dummy mock implementation. This can be replaced for your needs
```

Comment:
The mock implementation was the existing code (from your master branch) 
that I kept. I agree it does not follow the mockito scheme (which I 
don't really know about)

On 18/09/2015 17:27, Matias Meno wrote:
>
> In README.md 
> <https://github.com/exitlive/xcvbnm/pull/2#discussion_r39867224>:
>
> > +      String password = "my_pwd_1";
> > +      // Use the object (can allow mocking)
> > +      Xcvbnm xcvbnm = new Xcvbnm();
> > +      Result result = xcvbnm.estimate(password);
> > +      print(result.toMap());
> > +    }
> > +
> > +    main() {
> > +      test1();
> > +      test2();
> > +    }
> > +
> > +
> > +### mock
> > +
> > +Mock Xcvbnm can be mocked. there is a dummy mock implementation. This can be replaced for your needs
>
> I don't quite get your implementation of the mock object. If I want to 
> mock the object, I would use |mockito| 
> <https://github.com/fibulwinter/dart-mockito> to mock it like this:
>
> class  MockXcvbnmextends  Mockimplements  Xcvbnm {
>      noSuchMethod(Invocation invocation)=>  super.noSuchMethod(invocation);
> }
>
> There is no need to provide a mock implementation... what is it 
> supposed to do?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/exitlive/xcvbnm/pull/2/files#r39867224>.
>